apps/dashboard/app/views/layouts/application.html.erb

```diff
@@ -10,10 +10,11 @@
   <%= yield :head %>
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="turbolinks-root" content="/">
```

Comment:
I think turbolinks-root set to "/" is likely to broad - wouldn't all apps in OnDemand be affected by this? I'd expect you would at least need `/pun/sys/dashboard` i.e. the suburi or more appropriately here `<%= ENV['RAILS_RELATIVE_URL_ROOT'] || "/" %>`